docs/guides/privileged_devices.md

```diff
@@ -0,0 +1,146 @@
+
+#### Be aware of your responsibilities
+Familiarise yourself with Defra's information handling responsibilities, acceptable use policies and code of conduct for the use of IT and digital resources.
+
+We need to be especially mindful of these as the standard IT security controls often help us to comply with them without us even realising it.
+
+#### Understand the security principles
+Read the [CESG End User Devices Security Guidance](https://www.ncsc.gov.uk/guidance/end-user-device-security), in particular the [Security Principles](https://www.ncsc.gov.uk/guidance/end-user-devices-security-principles).
+
+This should give you a good understanding of the kind of controls that are required and why they are necessary.
+
+If you have any questions then speak to your manager.
+
+#### Ensure that your device stays patched and up to date
+The organisation will push regular patches and updates to your device according to the standard patching schedule. You should normally apply these immediately.
+
+The only exception to this is where you need to maintain older versions of software to do compatibility testing. In this case, you should update as soon as possible.
+
+#### Ensure that you are using official online resources
+One of the main reasons for privileged access is to be able to make use of online resources.
+However, this freedom of access can also be a threat that may cause devices to be compromised or leak official information.
+
+For this reason, only officially approved online resources should be accessed from unmanaged devices.
+
+In reality, any resource that you access for legitimate work is likely to be "officially approved", but you are responsible for all of your online activity and will be accountable for any incidents resulting from inappropriate access.
+
+If you are not sure whether an online resource is considered to be officially approved then speak to your manager.
+
+#### Use public networks securely
+You will need to be more careful than usual whenever you are connecting to public networks as the privileged access you have can represent a higher information security risk should your communications be intercepted.
+In particular, the captive portals typically found in cafes and hotels that require a username and password may not provide secure end-to-end encryption.
+
+Whenever you are connecting to a public network, you must ensure that you first evaluate whether it is safe to do so.
+
+#### Backup and secure your data
+Privileged access allows us to install our own tooling. One consequence of this is that we inevitably store more data locally.
+
+You must ensure that any unique content you create is backed up to secure, corporate repositories such as OneDrive or SharePoint.
+
+You should also consider backing up some of your local settings and data, which could help if your device fails and needs to be fixed or repalaced.
```

Comment:
```suggestion
You should also consider backing up some of your local settings and data, which could help if your device fails and needs to be fixed or replaced.
```

Reply:
😆 nice spot!

Reply:
I have a VS Code extension that spots spelling mistakes.

Doesn't help my awful grammar but at least it's always spelled correctly!